cloud/aws-functions/orphaned_eks_clusters.py

```diff
@@ -7,6 +7,11 @@
 from botocore.exceptions import ClientError
 from boto3.exceptions import Boto3Error
 
+def get_regions_list():
+    client = boto3.client('ec2')
+    regions = [region['RegionName'] for region in client.describe_regions()['Regions']]
+    return regions
```

Comment:
let's have this in a separate file (i.e `utils.py`) and define only one place then import in files

and you just `return [region['RegionName'] for region in client.describe_regions()['Regions']]`